galley/pkg/config/analysis/analyzers/gateway/gateway.go

```diff
@@ -80,7 +79,6 @@ func (*IngressGatewayPortAnalyzer) analyzeGateway(r *resource.Instance, c analys
 				}
 
 				service := rSvc.Message.(*v1.ServiceSpec)
-				// TODO I want to match service.Namespace to pod.ObjectMeta.Namespace
```

Comment:
line 78. service.Namespace is already matched with pod.ObjectMeta.Namespace.
so i remove TODO. i can update pr if i get it wrong